spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/service/impl/ComposedTaskProperties.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2017 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.dataflow.server.service.impl;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+/**
+ * Properties used to define the behavior of composed tasks created by
+ * Spring Cloud Data Flow.
+ *
+ * @author Glenn Renfro
+ */
+@ConfigurationProperties(prefix = ComposedTaskProperties.COMPOSED_TASK_PREFIX)
+public class ComposedTaskProperties {
+
+	public static final String COMPOSED_TASK_PREFIX = "spring.cloud.dataflow.composed.task";
```

Comment:
...or should we just have `TaskConfigurationProperties` with a prefix: `spring.cloud.dataflow.task`

then we could have a single property for now, called `composedTaskRunnerName`